src/ts/universes/level_2.ts

```diff
@@ -1,10 +1,22 @@
-import { GoalPlanet, Planet, UniversePlayer } from '../universe';
+import { GoalPlanet, Planet, UniversePlayer, UniverseText } from '../universe';
 import { buildUniverse, UniverseBuilder } from './builder';
 
 export const universe: UniverseBuilder = buildUniverse(() => ({
   objects: [
     new UniversePlayer({ x: 100, y: 100, orientation: Math.PI }),
-    new Planet({ x: 100, y: 100, texture: '#0f0', rotationSpeed: 0.3 }),
+    new Planet({ x: 100, y: 100, texture: '#0f0', rotationSpeed: 0.5 }),
     new GoalPlanet({ x: 300, y: 300, texture: '#f00', goalTexture: '#ff26' }),
+    new UniverseText({
+      x: 640,
+      y: 450,
+      fontSize: 32,
+      text: 'Some planets will rotate',
```

Comment:
I think the level itself makes this obvious, I don't think we should over explain things

Reply:
Eh, it can be very confusing as a user. I think over explaining provides a better experience than the risk of misunderstanding. Maybe they won't recognize the space as a 2d, top down view of space with a player.

Reply:
Isn't this the level where you start out on a rotating planet? Feels like how people complain that newer Zelda games over explain things. The players are intelligent, they can see that the planet is rotating. Gold level design doesn't require an explanation.